### PR TITLE
Feature/quote form

### PR DESCRIPTION
**Pull request by hanna446**
I updated design based on Zeplin.

### CODE REVIEW

File: js/quote_tool.js [hunk 1]
```diff
@@ -1,64 +1,245 @@
...
+  }
+};
+/*
+ * This code assign the calendar to "txtNumNigth" and gives it a date format.
+ */
+flatpickr("#txtNumNigth", {
+  mode: "range",
+  dateFormat: "F, d, Y "
+});
+//   minDate: "1920-01-01",
```
Comment (camigomezdev) [inline js/quote_tool.js]: please, don't send code commented.

**Comment by camigomezdev** [comment]
Half of the code has a BEM structure, check that all classes have the same structure.

**Comment by jhonatanrojasbastidas** [comment]
En el número de huéspedes debería limitarse .

**Comment by jhonatanrojasbastidas** [comment]
> En el número de huéspedes debería limitarse .

Se podría manejar como un improvement.

**Review by jhonatanrojasbastidas — APPROVED** [review]

File: pages/quote.html [hunk 2]
```diff
@@ -4,26 +4,263 @@
...
+        <!-- details -->
+        <div class="div__details">
+          <h3 class="details">Detalles</h3>
+          
+          <div class="div__field">
+          <input
+            type="text"
+            id="txtNumPeople"
+            class="form__input"
+            onblur="validationNumPerson(this)"
```
Comment (camigomez35) [inline pages/quote.html]: please remember use only eventListener for events

File: pages/quote.html [hunk 3]
```diff
@@ -4,26 +4,263 @@
...
+            
+        </p>
+        <div class="div__quote">
+          <!--div__field-->
+          <div class="div__field">
+            <input
+              type="text"
+              id="input-name"
+              class="form__input"
+              onblur="validationName(this)"
```
Comment (camigomezdev) [inline pages/quote.html]: Please use `addEventListener` instead of inline events (i.e. `onblur`)

**Comment by josemariogutierrez** [comment]
- The line on the input is not aligned to the text. 
- Blue color when focused is not the same that the one used in Zeplin.
- `Enviar a mi Correo` button should be centered. 
- Please don't use percentage to place the alert icon. 
- Desktop view still needs to be implemented.

File: index.html [hunk 4]
```diff
@@ -1,5 +1,16 @@
 <!DOCTYPE html>
 <html lang="es">
+  <head>
```
Comment (camigomezdev) [inline index.html]: Please don't send this file... Head is duplicated.

File: pages/quote.html [hunk 5]
```diff
@@ -4,26 +4,309 @@
...
+              type="text"
+              id="input-phone"
+              class="form-field__input"
+              onblur="validationPhone(this)"
+            />
+            <label for="" class="form-field__label">Teléfono</label>
+            <div id="phone-error" class="form__message--error"></div>
+          </div>
+          <!--form__field-END-->
+          <div class="form__ckeckbox">
```
Comment (camigomezdev) [inline pages/quote.html]: typo -> `form__checkbox`

File: pages/quote.html [hunk 6]
```diff
@@ -4,26 +4,309 @@
...
+            type="text"
+            id="txtNumNigth"
+            class="form-field__calendar form-field__input"
+          />
+          <label for="" class="form-field__label">Número de noches</label>
+        </div>
+        <!--details-END-->
+        <!--options__transport-->
+        <div class="form__options">
+          <h5 class="form-options-subtitle">Transporte</h5>
```
Comment (camigomezdev) [inline pages/quote.html]: `form-options__subtitle`

**Comment by camigomezdev** [comment]
please change the labels when it's ready

**Review by camigomezdev — CHANGES_REQUESTED** [review]
`onblur="validationName(this)"` could be to `eventListener`

File: .vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (jusk02) [inline .vscode/settings.json]: please remove this file

File: css/quotes.css [hunk 8]
```diff
@@ -1,185 +1,271 @@
...
+  overflow-x: hidden;
 }
 
-.form {
-  margin-left: 5%;
-  margin-right: 5%;
+.form__container {
+  /* margin-left: 5%;
+  margin-right: 5%; */
   max-width: 90%;
```
Comment (jusk02) [inline css/quotes.css]: please remove commented rules, also if in the HTML you are using the generic `container` helper, what is the purpose of setting `max-width: 90%;`? where did the value of 90% come from?

**Comment by jusk02** [comment]
"Cotización enviada con éxito ✔" appears on page load, this should only appear after a successful submission

File: js/quote_tool.js [hunk 9]
```diff
@@ -34,31 +192,127 @@
...
+    document.getElementById("txtErrors").appendChild(errorList);
+  }
+};
+
+// validate the opted data to verify the number of tours and nights.
+const validateData = (form) => {
+  var errors = [];
+  if (form.numberOfNigth < (getTrues(form.tour) + getTrues(form.hike))) {
+    errors.push(
+      "el numero seleccionado de dias es poco para la suma de tours y o caminatas, cada evento toma un dia."
```
Comment (Sonny97) [inline js/quote_tool.js]: I reckon it sounds better if we say it like this:
"El número seleccionado de días es poco para la suma de tours y/o caminatas acumuladas, ten en cuenta que cada evento toma un día."

File: css/quotes.css [hunk 10]
```diff
@@ -1,185 +1,271 @@
...
 
 /*
-* In this code insert the figure "✓" after that a checkbox is selected; the figure is created
-* assigning attributes and properties to the content.
-*/
-.checkbox__container input:checked + .checkmark:after {
+ * In this code insert the figure "✓" after that a checkbox is selected; the figure is created
+ * assigning attributes and properties to the content.
+ */
+.form-options__label input:checked + .form-options__checkmark:after {
```
Comment (camigomezdev) [inline css/quotes.css:120]: please don't use tags like `input` for giving class...

Comment (jusk02) [reply css/quotes.css:120]: como esto puede ser algo confuso...

Por favor no usen tags para asignar apariencia, en este caso `input`, recurran a las clases unicamente.

si esto aun es confuso escribanme.

File: pages/buttons.html [hunk 11]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (camigomezdev) [inline pages/buttons.html]: please remove this file

File: js/readme.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
+names of the fields for the called by the id from the html are :
```
Comment (jusk02) [inline js/readme.md]: Please make this documentation file not so ugly.

Reference this to polish it up
https://github.com/ricval/Documentacion/blob/master/Guias/GitHub/mastering-markdown.md

File: css/quotes.css [hunk 13]
```diff
@@ -1,185 +1,269 @@
...
-.form__title {
-  font-size: 22px;
-  font-weight: bold;
-  text-align: justify;
-  margin-top: 50px;
-  margin-bottom: 10px;
-  color: rgba(0, 0, 0, 0.7);
+.form__container {
+  max-width: 100%;
+  padding: 0 20px 0 20px;
```
Comment (jsebax) [inline css/quotes.css]: This can be written as `padding: 0 20px;`

File: css/quotes.css [hunk 14]
```diff
@@ -1,185 +1,269 @@
...
 }
 
-.form__subtitles {
-  font-weight: bold;
-  text-align: justify;
-  margin-bottom: 18px;
-  color: rgba(0, 0, 0, 0.7);
+.details {
```
Comment (jsebax) [inline css/quotes.css]: Is this class being used? I don't see it in the markup

File: js/quote_tool.js [hunk 15]
```diff
@@ -1,3 +1,152 @@
...
+ */
+flatpickr("#txtNumNigth", {
+  mode: "range",
+  dateFormat: "F, d, Y ",
+  onChange: function(selectedDates) {
+    let totalDays = Math.round(
+      (selectedDates[1] - selectedDates[0]) / (1000 * 60 * 60 * 24)
+    );
+    document.getElementById("numNigth").value = totalDays;
+    activateBoton();
```
Comment (jusk02) [inline js/quote_tool.js]: no spanglish please `activateButton` ;)

File: js/quoting_readme.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+
+## Names of the fields for the called by the id from the html are :
+
+**Transport = ts** 🚙
+ * *_Transporte Medellin Necocli_* = `<id>` = "ts1"
+ * *_Transporte Necocli Capurgana_* = `<id>` = "ts2"
+ * *_Transporte Capurgana Necocli_* = `<id>` = "ts3"
+ * *_Transporte Necocli Medellin_* = `<id>` = "ts4"
+
+**food = f** 🍔
```
Comment (jusk02) [inline js/quoting_readme.md:11]: breakfast , dinner

**Comment by jusk02** [comment]
Repeating comment since it was totally ignored:
"
"Cotización enviada con éxito ✔" appears on page load, this should only appear after a successful submission
"

**Review by camigomezdev — APPROVED** [review]

**Pull request merged**